test/aws-cur/cost-report.test.ts

```diff
@@ -112,4 +112,14 @@ describe('CostReport', () => {
     template.resourceCountIs('AWS::CUR::ReportDefinition', 2);
   });
 
+  test('regions other than us-east-1', () => {
+    const regionOtherStack = new Stack(app, 'OtherRegionStack', { env: { region: 'ap-northeast-1' } });
+
+    expect(() => new CostReport(regionOtherStack, 'MyCustomCostReport', {
+      costReportName: 'custom-cur',
+      reportGranularity: ReportGranularity.DAILY,
+      format: CurFormat.PARQUET,
+    })).toThrow('CUR as it is only available in the us-east-1 region');
```

Comment:
Match to the comment above.

Reply:
Modified to align with the implementation.